apps/etterlatte-behandling/src/main/kotlin/behandling/GenerellBehandlingService.kt

```diff
@@ -293,6 +291,36 @@ class RealGenerellBehandlingService(
         }
     }
 
+    override fun oppdaterVirkningstidspunkt(
+        behandlingId: UUID,
+        dato: YearMonth,
+        ident: String,
+        begrunnelse: String
+    ): Virkningstidspunkt {
+        val behandling = hentBehandling(behandlingId) ?: run {
+            logger.error("Prøvde å oppdatere virkningstidspunkt på en behandling som ikke eksisterer: $behandlingId")
+            throw RuntimeException("Fant ikke behandling")
+        }
+
+        val virkningstidspunkt = Virkningstidspunkt.create(dato, ident, begrunnelse)
+        try {
+            behandling.oppdaterVirkningstidspunkt(virkningstidspunkt)
+                .also {
+                    inTransaction {
+                        behandlinger.lagreNyttVirkningstidspunkt(behandlingId, virkningstidspunkt)
+                        behandlinger.lagreStatus(it)
+                    }
+                }
+        } catch (e: NotImplementedError) {
+            logger.error(
```

Comment:
Ville tenkt at det er betre her å pakke inn e saman med loggteksten, for å få det ut samla i loggen, men er nok på grensa til pirk.